custom/icds_reports/utils/__init__.py

```diff
@@ -231,12 +231,13 @@ def _custom_data(self, selected_location, domain):
                 loc_type = selected_location.location_type.name
             else:
                 loc_type = None
+            tags = ["location_type:{}".format(loc_type), "report_slug:{}".format(self.slug)]
+            if allow_conditional_agg:
+                tags.append("allow_conditional_agg:yes")
             datadog_histogram(
                 "commcare.icds.block_reports.ucr_querytime",
                 timer.duration,
-                tags="config:{}, location_type:{}, report_slug:{}".format(
```

Comment:
was it intentional to leave config out of this?

Reply:
Thanks for catching, was an oversight!